frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx

```diff
@@ -299,6 +317,25 @@ export const LogDetails: React.FC<Props> = ({
 							</Box>
 						</LinkButton>
 					)}
+					{row.original.node.traceID &&
+						row.original.node.source === 'backend' && (
```

Comment:
Added this check since we do not record frontend traces - there is a few cases that we do not write a [trace](https://github.com/highlight/highlight/blob/main/backend/otel/otel.go#L198)
- metric events
- logging events

I did notice that we still add a trace_id and span_id for logs [in handleTrace](https://github.com/highlight/highlight/blob/main/backend/otel/otel.go#L265-L266) and [in handleLog](https://github.com/highlight/highlight/blob/main/backend/otel/otel.go#L439-L440). I'm not sure which one, but should we remove this link from one since the trace may not exists? Or could there be a outer trace used?

Reply:
It seems like the log may still be linked to a previously created trace, but we do not want to create a span for this log. This may result in some logs appearing to have a related trace, but none in fact, if it is the only span in the trace 🤔

Reply:
believe it should be fine to stop writing a `trace id` or a `span id` for logs when they do not have one, rather than creating one. perhaps there is something else we can use to remember that this is a trace-less log record though?